fe/fe-core/src/main/java/com/starrocks/leader/Checkpoint.java

```diff
@@ -233,7 +233,7 @@ private boolean replayAndGenerateStarMgrImage(long logVersion) {
         try {
             return starMgrServer.replayAndGenerateImage(imageDir, logVersion);
         } catch (Exception e) {
-            LOG.error("Exception when generate new star mgr image file, {}.", e.getMessage());
+            LOG.error("Exception when generate new star mgr image file", e);
```

Comment:
intend to do the `{}, e.getMessage()` to avoid printing long long stack trace.

Reply:
When I encountered this issue, there was little useful information in fe.warn.log, with only a null present. This is not conducive to troubleshooting the problem.